client/question/components/LearnerViewDirective.js

```diff
@@ -98,8 +86,18 @@ tie.directive('learnerView', [function() {
                   Submit for Grading
                 </button>
                 <button class="tie-run-button tie-button protractor-test-run-code-button" ng-class="{'tie-button-green': !pageIsIframed}" ng-click="submitCode(editorContents.code)" ng-disabled="SessionHistoryService.isNewBalloonPending()" title="Click anytime you want feedback on your code">
-                  Get Feedback
+                  RUN
                 </button>
+                <div>
+                  <select class="tie-select-menu protractor-test-submission-select"
+                      ng-change="changeSubmission(currentSubmissionNumber)"
```

Comment:
changeSubmission is a bit of a odd name for the method, but let's talk a bit more about this. I think it would be best if the method name were more evocative of the functionality, which is pulling an older submission out of its history. Any thoughts on a name?